operator/api/v1alpha1/keptnapp_types.go

```diff
@@ -37,6 +37,7 @@ type KeptnAppSpec struct {
 
 // KeptnAppStatus defines the observed state of KeptnApp
 type KeptnAppStatus struct {
+	CurrentVersion string `json:"currentVersion,omitempty"`
```

Comment:
why is this needed ? We already have version in the .spec section which represents the current version of the appVersion

Reply:
This is overwritten with the next app update, therefore we write it to the status as this is preserved when the new app definition is applied …